core/loadpoint.go

```diff
@@ -303,7 +304,7 @@ func (lp *Loadpoint) restoreSettings() {
 	if v, err := lp.settings.String(keys.Mode); err == nil && v != "" {
 		lp.setMode(api.ChargeMode(v))
 	}
-	if v, err := lp.settings.Int(keys.PhasesConfigured); err == nil && v > 0 {
```

Comment:
Wenn ich Phasenumschaltung habe sollte ich in der Config 0 hinterlegen können dürfen. Der Change hier wäre mit dem neuen Default 3 falsch?

Reply:
Verstehe den Punkt noch nicht ganz. 0 ist bei Phases ein valider Wert in der Datenbank. Habe ich auf phases=0 gestellt möchte ich auch, dass das wieder so geladen wird. Daher hab ich den >0 Check entfernt.
Oder hab ich die Settings Methode falsch verstanden. Gehen davon aus, dass sie im Fall "kein Setting existiert" nen Error zurückliefert und wie hier nur ins if laufen wenn es vorher eine explizite Einstellung durch den Nutzer gab.

Reply:
Ahhh sorry. Ich dachte du hast ihn *hinzu* gefügt. Perfekt! Sollten wir aber *nur* erlauben wenns Phasenumschaltung gibt. Also innen noch eine Prüfung rein? Der private Setter validiert nämlich nicht mehr.

Reply:
Hm, ich hab die Prüfung jetzt noch auch in den privaten Setter eingebaut. Allerdings nur mit return+warn, um ungültige Zustände zu vermeiden. Im Normalbetrieb sollte das eigentlich nicht vorkommen. Kann aber bei Gerätewechsel / Versionsupdate oder so durchaus passieren.

Reply:
Aus Konsistenzgründen ungern. Können wir das hier prüfen?

Reply:
Mach gerne nen alternativen Vorschlag. Lieber `setConfiguredPhases` nen `error` returnen lassen und den im Public abfragen?

Reply:
Hier prüfen und einfach nicht aufrufen.